### DISCUSSION TITLE
Debugger "process failed to start"

### DISCUSSION THREAD

**Discussion by ghost** [orbitalquark/textadept#341] [Q&A]
Hi.
Yesterday I was happily debugging my Lua script.
Today I receive an error buffer with the following message.
I don't know what might have changed.
Any help will be greatly appreciated.

.../jeff/.local/opt/textadept/modules/debugger/lua/init.lua:174: 
lua -e "io.stdout:setvbuf('no')"
-e "package.path = 
package.path .. ';
/home/jeff/.local/opt/textadept/modules/debugger/lua/?.lua;
/home/jeff/.textadept/modules/debugger/lua/?.lua'"
-e "require('mobdebug').start()"
"/home/jeff/test.lua": process failed to start
Debugger started

**Comment by orbitalquark** [comment]
I've noticed this too, on occasion. Restarting Textadept might resolve 
this. It's on my TODO list to look into.

**Reply by ghost** [reply]
No such luck.
Will try again tomorrow.
Thanks for letting me know I'm not alone.
